listen_test.go

```diff
@@ -372,17 +373,49 @@ func Test_Listen_Master_Process_Show_Startup_MessageWithAppName(t *testing.T) {
 	require.True(t, strings.Contains(startupMessage, app.Config().AppName))
 }
 
+// go test -run Test_Listen_Master_Process_Show_Startup_MessageWithAppNameNonAscii
+func Test_Listen_Master_Process_Show_Startup_MessageWithAppNameNonAscii(t *testing.T) {
+	cfg := ListenConfig{
+		EnablePrefork: true,
+	}
+
+	appName := "Serveur de vérification des données"
+	app := New(Config{AppName: appName})
+
+	startupMessage := captureOutput(func() {
+		app.startupMessage(":3000", false, "", cfg)
+	})
+	fmt.Println(startupMessage)
```

Comment:
why would you keep fmt.Println ?

Reply:
> why would you keep fmt.Println ?

We plan to remove most of them later to make testing messages cleaner.